plugins/inputs/system/DISK_README.md

```diff
@@ -51,3 +51,57 @@ In this case, the host's root volume should be mounted into the container and th
 > disk,fstype=autofs,path=/net free=0i,inodes_free=0i,inodes_total=0i,inodes_used=0i,total=0i,used=0i,used_percent=0 1453832006274157077
 > disk,fstype=autofs,path=/home free=0i,inodes_free=0i,inodes_total=0i,inodes_used=0i,total=0i,used=0i,used_percent=0 1453832006274169688
 ```
+
+
+# DiskIO Input Plugin
+
+The diskio input plugin gathers metrics about disk traffic and timing.
+
+### Configuration:
+
+```
+# Read metrics about disk IO by device
+[[inputs.diskio]]
+  ## By default, telegraf will gather stats for all devices including
+  ## disk partitions.
+  ## Setting devices will restrict the stats to the specified devices.
+  # devices = ["sda", "sdb"]
+  ## Uncomment the following line if you need disk serial numbers.
+  # skip_serial_number = false
+```
+
+Data collection is based on github.com/shirou/gopsutil. This package handles platform dependencies and converts all timing information to miliseconds.
+
+
+### Measurements & Fields:
+
+- diskio
+    - reads (integer, counter)
+    - writes (integer, counter)
+    - read_bytes (integer, bytes)
+    - write_bytes (integer, bytes)
+    - read_time (integer, miliseconds)
+    - write_time (integer, miliseconds)
+    - io_time (integer, miliseconds)
+    - iops_in_progress (integer, miliseconds) (since #2037, not yet in STABLE)
```

Comment:
Fixed, thanks. I only have STABLE installed, so unable to test iops_in_progress.